packages/io-ts-numbers/src/ZeroInt.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @since 0.1.0
+ */
+
+import * as t from 'io-ts'
+
+import { Int } from './Int'
+import { Zero } from './Zero'
+
+/**
+ * @since 0.1.0
+ */
+export const ZeroInt = t.intersection([Zero, Int], 'ZeroInt')
```

Comment:
It strikes me that we can encode the knowledge the `zero` is always an integer by moving this intersection to the existing `Zero.ts` file